### PR TITLE
Remove old wordcount

### PR DESCRIPTION
**Pull request by phlax**
seems like this was evernote specific - we can readd if we feel its useful

### CODE REVIEW

File: docs/server/settings.rst [hunk 1]
```diff
@@ -501,7 +501,6 @@ Translation environment configuration settings.
   Current options:
 
   - Translate Toolkit (default) - translate.storage.statsdb.wordcount
-  - Pootle - pootle.core.utils.wordcount.wordcount
```
Comment (unho) [inline docs/server/settings.rst]: I would remove `POOTLE_WORDCOUNT_FUNC` setting altogether if we are dropping this function.

Comment (ta2-1) [inline docs/server/settings.rst]: I think it is a valid pattern to overwrite wordcount function. probs it can be implemented using getter.

Comment (phlax) [inline docs/server/settings.rst]: yep agreed re getter

**Comment by unho** [comment]
lgtm

**Pull request merged**